### DISCUSSION TITLE
Lesson 6: KeyError: 'ganache-local'  TimeStamp 5:42:00

### DISCUSSION THREAD

**Discussion by santoroeth** [smartcontractkit/full-blockchain-solidity-course-py#1223] [Q&A] [answered]
Hi, I hope everyone is doing well.

I have the following error when I try to use the local-ganache network:

Running 'scripts\deploy.py::main'...
The active network is ganache-local
Deploying Network...
Mock Deployed!
  File "C:\Users\JuanGuillermo\AppData\Local\Programs\Python\Python310\lib\site-packages\brownie\_cli\run.py", line 49, in main
    return_value, frame = run(
  File "C:\Users\JuanGuillermo\AppData\Local\Programs\Python\Python310\lib\site-packages\brownie\project\scripts.py", line 103, in run
    return_value = f_locals[method_name](*args, **kwargs)
  File ".\scripts\deploy.py", line 42, in main
    deploy_fund_me()
  File ".\scripts\deploy.py", line 28, in deploy_fund_me
    fund_me=FundMe.deploy(price_feed_address,{"from":account}, publish_source=config["networks"][network.show_active()].get("verify"))
**KeyError: 'ganache-local'**

**This is my "helpful_scripts.py"**
![image](https://user-images.githubusercontent.com/97750911/157244999-0af80679-c93d-4413-a380-2d9bbc5e9791.png)

**This is my "brownie-congif.yaml"**
![image](https://user-images.githubusercontent.com/97750911/157245120-fcb252fe-faf1-40bf-85f7-e8b1d03fd404.png)

**This is my "ganache UI"**
![image](https://user-images.githubusercontent.com/97750911/157245284-bcb2a1b1-ad85-483e-8c27-9923f6615574.png)

**And this is how my networks list look like:**
![image](https://user-images.githubusercontent.com/97750911/157246376-f311f46e-3703-40b9-9095-7b221de77d1f.png)

**hope can help me, thank you very much and greetings from Colombia!!**

**Comment by cromewar** [comment, marked as answer]
Hello @santoroeth 
You have an indentation error on your `brownie-config.yaml`
 
![image](https://user-images.githubusercontent.com/20199180/157250129-267bf6e8-b79c-4d81-af35-3091e98e096c.png)
 this should be one space to the left.

**Comment by santoroeth** [comment]
Hello @cromewar 

Thank you very much, I tried everything and I had not seen this simple little error.

Greetings and thanks again

**Reply by cromewar** [reply]
I'm glad is solved now

**Reply by santoroeth** [reply]
Believe me, I'm even more glad

**thanks hawk eye!**
